tests/ai/src/stateful-prompts.ts

```diff
@@ -0,0 +1,62 @@
+import { ChatPrompt, IChatModel, Message } from '@microsoft/teams.ai';
+import { ActivityLike, IMessageActivity, MessageActivity } from '@microsoft/teams.api';
+
+// :snippet-start: stateful-prompts-state-initialization
+// Simple in-memory store for conversation histories
+// In your application, it may be a good idea to use a more
+// persistent store backed by a database or other storage solution
```

Comment:
Should this be above the snippet? Or included?

Reply:
I think included